pytpcc/tpcc.py

```diff
@@ -53,6 +54,24 @@
     '%(asctime)s [%(funcName)s:%(lineno)03d] %(levelname)-5s: %(message)s'))
 logging.getLogger('').addHandler(console)
 
+NOTIFY_PHASE_START_PATH = '/data/workdir/src/flamegraph/notify_phase_start.py'
+NOTIFY_PHASE_END_PATH = '/data/workdir/src/flamegraph/notify_phase_start.py'
+
+## ==============================================
+## noftifyDsiOfPhaseStart
+## ==============================================
+def noftifyDsiOfPhaseStart(phasename):
+    if os.path.isfile(NOTIFY_PHASE_START_PATH):
+        call(["python3", NOTIFY_PHASE_START_PATH, phasename])
```

Comment:
I think https://docs.python.org/3/library/subprocess.html#subprocess.run is the recommended way to run subprocess now. `call` is listed as an older API. Can we use run? Do we have Python newer than 3.5

Relatedly, do we want to check the status code of this and throw an exception? Run has a "check" parameter that can be used for that.

Reply:
I've swapped to `subprocess.run` and checked for the error code. It's quite neat how `subprocess.run` returns a human-readable object that contains all relevant information:
```
CompletedProcess(args=['ls', '-l', '/dev/null'], returncode=0,
stdout=b'crw-rw-rw- 1 root root 1, 3 Jan 23 16:23 /dev/null\n', stderr=b'')
```
Patch for the new version:
https://spruce.mongodb.com/version/65a7b01e7742ae630812f6ec